docs/competition/teams/process-docs/overnight-battery-loan.md

```diff
@@ -0,0 +1,13 @@
+# Overnight Battery Loan
+
+Teams often request a battery and charger for use overnight to test their robots.
+
+## Obtaining batteries
+
+Batteries and chargers should be collected from helpdesk by teams. Teams should be given a charger, and no more than 2 batteries.
+
+The teams TLA, and the part codes of the batteries and changer should be recorded on the [Overnight battery loan]() form.
+
+## Returning batteries
+
+Batteries should be returned to reception on Sunday when teams enter the venue. The list of teams who took batteries overnight should be used to confirm which teams bring them back.
```

Comment:
What enforces this? Are we relying on their honesty, or is this something for reception to do? If the latter
* how does reception know which teams to collect batteries from?
* we should modify the reception docs to account for this and to ensure they have boxes for collecting the kit into on the Sunday morning

Reply:
> how does reception know which teams to collect batteries from

The list of teams who took batteries overnight should be used to confirm which teams bring them back.

> we should modify the reception docs

Agreed.

Reply:
Ah, ok. Could we be explicit about that?
``` suggestion
Batteries should be returned to reception on Sunday when teams enter the venue.
Once all the teams have left on Saturday, the list of teams who took batteries overnight should be passed to reception who will ensure teams return their batteries on the Sunday morning.
```

Hrm. If we do pass the list over to reception, how does the helpdesk then know which items the team had overnight?
Would it make sense to instead provide reception with a _separate_ list of TLAs, perhaps _copied_ from the list of loans?

Reply:
I see what you mean about the quoted sentence, however I find that very easy to read as being about the Saturday and it seems easy to overlook that it applies to the Sunday morning too.
There's also no mention in the reception docs about checking teams who've borrowed batteries overnight and directing them to the arrivals desk to return those items.